code/modules/mob/living/carbon/superior_animal/human/subtype/kriosan/kriosan.dm

```diff
@@ -0,0 +1,93 @@
+/mob/living/carbon/superior_animal/human/rebel
```

Comment:
also this is just personal preference but i usually don't like using base types in any capacity that isn't for holding vars, it makes things messy when i want to make a change to this specific type but none else